src/Task/TaskRunner.php

```diff
@@ -19,7 +19,7 @@ public function run (string $task, SymfonyStyle $io) : bool
     {
         $io->section("Run: `{$task}`");
 
-        $process = Process::fromShellCommandline($task);
+        $process = Process::fromShellCommandline($task, null, null, null, null);
```

Comment:
```suggestion
        $process = Process::fromShellCommandline($task);
        $process->setTimeout(null);
```

Instead of adding 20 `null` (which is quite unreadable), can we instead just add a single call to the setter?